src/java/com/phenix/pct/ABLUnit.java

```diff
@@ -92,8 +92,10 @@ public void addConfiguredFileset(FileSet set) {
         testFilesets.add(set);
     }
 
-    private void writeJsonConfigFile() throws IOException {
+    private void writeJsonConfigFile() throws IOException, RuntimeException {
         JsonWriter writer = null;
+        String foobar = "abc";
+        String foobar2 = "def";
         try {
             writer = new JsonWriter(new FileWriter(json));
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Close this "FileWriter". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS2095)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this use of constructor "FileWriter(File)" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1943)